tensorflow/core/kernels/fused_batch_norm_op.cc

```diff
@@ -54,25 +54,20 @@ struct FusedBatchNorm<CPUDevice, T, U> {
                   Tensor* batch_var_output, Tensor* saved_mean_output,
```

Comment:
In case you missed it before: Also add tests to normalization_test.py, layers_test.py, and CPU tests to nn_fused_batchnorm_test.py.

Reply:
Sorry, I had missed it. Tests included in latest commit.